### PR TITLE
Improve mobile help menu styles.

### PR DESCRIPTION
**Pull request by tofumatt**
## Summary

Addresses issue #2846.

## Relevant technical choices

This hides the text on mobile to make sure the issues mentioned in https://github.com/google/site-kit-wp/issues/2846#issuecomment-804036559 don't come up. It looks like this:

Screenshot 2021-03-22 at 21 09 27

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/sass/components/global/_googlesitekit-help-menu.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin .googlesitekit-help-menu {
+
+	margin-right: $grid-gap-phone / 2;
+	padding: 0;
+	width: 52px;
```
Comment (aaemnnosttv) [inline assets/sass/components/global/_googlesitekit-help-menu.scss:23]: Can you elaborate on why this is needed? Seems like an arbitrary fixed value.

Also, are there really any styles which are specific to the help menu? I would all 3 menus should largely share the same styles.

**Review by aaemnnosttv — CHANGES_REQUESTED** [review]
Thanks @tofumatt – this looks good for the most part, just one question for you.

Looks like VRT needs to be updated as well (currently failing):
image

File: assets/sass/components/global/_googlesitekit-help-menu.scss (hunk 1 above)
Comment (tofumatt) [reply assets/sass/components/global/_googlesitekit-help-menu.scss:23]: Actually, you're right, it's no longer needed 👍🏻

**Comment by tofumatt** [comment]
VRTs updated and that extra width is removed. Should be good-to-go now 😃

**Review by aaemnnosttv — APPROVED** [review]
LGTM!

**Pull request merged**